### PR TITLE
allow cluster up/tsb to tolerate any version of 3.7.x, prerelease or otherwise

### PR DESCRIPTION
**Pull request by bparees**
fixes https://bugzilla.redhat.com/show_bug.cgi?id=1488060

### CODE REVIEW

**Comment by bparees** [comment]
@csrwng ptal

File: pkg/oc/bootstrap/docker/up.go [hunk 1]
```diff
@@ -1061,7 +1048,7 @@ func (c *ClientStartConfig) InstallTemplateServiceBroker(out io.Writer) error {
 	}
 	// TODO we want to use this eventually, but until we have our own image for TSB, we have to hardcode this origin
 	//return c.OpenShiftHelper().InstallTemplateServiceBroker(f, c.imageFormat())
-	return c.OpenShiftHelper().InstallTemplateServiceBroker(f, c.Image, c.ServerLogLevel)
+	return c.OpenShiftHelper().InstallTemplateServiceBroker(f, c.Image+":"+c.ImageVersion, c.ServerLogLevel)
```
Comment (csrwng) [inline pkg/oc/bootstrap/docker/up.go]: fmt.Sprintf?

Comment (bparees) [reply pkg/oc/bootstrap/docker/up.go]: updated.

**Comment by bparees** [comment]
reapplied lgtm post rebase.

**Comment by bparees** [comment]
/retest

Ben Parees | OpenShift

**Pull request merged**